### PR TITLE
Modified version TAP98

### PR DESCRIPTION
**Pull request by juandosimple**
- [U] Updated the version of the scripts to match those of the plugin already on the server, in order to avoid conflicts.
- [M] Modified these three files so that it doesn't dynamically search for a library but instead always has the interactive video library pre-selected 'H5P.InteractiveVideo 1.24'.

### CODE REVIEW

File: scripts/h5peditor-av.js [hunk 1]
```diff
@@ -318,11 +318,14 @@ H5PEditor.widgets.video = H5PEditor.widgets.audio = H5PEditor.AV = (function ($)
...
-    // Check if source is provider (Vimeo, YouTube, Panopto)
-    const isProvider = file.path && C.findProvider(file.path);
+    // Check if source is YouTube
+    var youtubeRegex = C.providers.filter(function (provider) {
+      return provider.name === 'YouTube';
+    })[0].regexp;
+    var isYoutube = file.path && file.path.match(youtubeRegex);
 
     // Only allow single source if YouTube
-    if (isProvider) {
```
Comment (schup) [inline scripts/h5peditor-av.js:325]: What is the purpose of this change? Why do we limit it to youtube?

File: scripts/h5peditor-library-selector.js [hunk 2]
```diff
@@ -247,7 +235,9 @@ ns.LibrarySelector.prototype.loadSemantics = function (library, params, metadata
   this.$selector.attr('disabled', true);
 
   ns.resetLoadedLibraries();
-  ns.loadLibrary(library, function (semantics) {
+  // ns.loadLibrary(library, function (semantics) {
+  ns.loadLibrary("H5P.InteractiveVideo 1.24", function (semantics) {
```
Comment (schup) [inline scripts/h5peditor-library-selector.js:239]: IMHO not great to repeat the same string - including a chaning version number - so many times.

File: scripts/h5peditor-av.js (hunk 1 above)
Comment (juandosimple) [reply scripts/h5peditor-av.js:325]: Not sure, that comes with the plugin, i was matching the version that we have (uploaded in the server) with the version that the respository has!

File: scripts/h5peditor-library-selector.js (hunk 2 above)
Comment (juandosimple) [reply scripts/h5peditor-library-selector.js:239]: Maybe i can create a variable that replaces the library!

File: scripts/h5peditor-av.js (hunk 1 above)
Comment (schup) [reply scripts/h5peditor-av.js:325]: IMHO our starting point should be the repo. 
The changes look like they are coming from an older version.
@MerqMercer what are your thoughts?

Should we replace "H5P.InteractiveVideo" with "Tapybl.InteractiveVideo" to avoid problems in the future?

Comment (juandosimple) [reply scripts/h5peditor-av.js:325]: @schup Should we fork from the last version ?

Comment (schup) [reply scripts/h5peditor-av.js:325]: I believe we are up-to-date on the master branch.
image

Comment (juandosimple) [reply scripts/h5peditor-av.js:325]: Ahh, I see now. This one is the latest version, but the actual plugin is in version 1.5.4. So the plugin is not up to date with this submodule. Perhaps we should update the plugin as well?
